### ISSUE TITLE
set resource constraints on sso-dashboard pods

### ISSUE THREAD

**Issue by danielhartnell** [mozilla-iam/eks-deployment#35]
Best practice to avoid a pod consuming more resources than anticipated.

**Comment by danielhartnell** [comment]
I'm taking a guess at the resources to provide to the containers. This seems like a reasonable start. I was surprised to see about 700 megabytes of memory consumed by the Flask application that's currently being tested in Kubernetes. For the original development site, I was able to get CPU usage metrics but it does not look like memory metrics are available.

```
          resources:
            requests:
              memory: "768Mi"
              cpu: "250m"
            limits:
              memory: "1024Mi"
              cpu: "500m"
```

Once we have monitoring in Prometheus, we can use that feedback loop to make adjustments.

**Comment by danielhartnell** [comment]
Fixed by #46
